tests/gui/widgets/test_docks.py

```diff
@@ -11,22 +11,25 @@
 )
 
 
-def test_videos_dock(qtbot, centered_pair_predictions: Labels,
+def test_videos_dock(
+    qtbot,
+    centered_pair_predictions: Labels,
     small_robot_mp4_vid: Video,
     centered_pair_vid: Video,
-    small_robot_3_frame_vid: Video,):
+    small_robot_3_frame_vid: Video,
+):
     """Test the `DockWidget` class."""
 
     # Add some extra videos to the labels
     labels = centered_pair_predictions
-    labels.add_video(small_robot_mp4_vid)
-    labels.add_video(centered_pair_vid)
     labels.add_video(small_robot_3_frame_vid)
+    labels.add_video(centered_pair_vid)
+    labels.add_video(small_robot_mp4_vid)
     assert len(labels.videos) == 4
 
     # Create the dock
     main_window = MainWindow()
-    main_window.labels = labels
+    main_window.commands.loadLabelsObject(labels)
```

Comment:
Since we spent some time troubleshooting this, it's worth adding a comment as to why we're using it via the command as opposed to just setting the attribute for future reference in case we need to write these types of tests again.

Reply:
sure, I'll add a comment